services/movie.py

```diff
@@ -15,13 +16,17 @@ def get_movies(
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
 
 
+@transaction.atomic
 def create_movie(
     movie_title: str,
```

Comment:
According to the code style guide, you should use double quotes instead of single quotes. It would be better to use double quotes for string values to keep the code style consistent.